README.md

```diff
@@ -89,7 +89,7 @@ const baseStyle = {
   width: "50%"
 };
 
-const SvelteInReact = reactSvelte(SvelteApp, baseStyle, "div");
+const SvelteInReact = toReact(SvelteComponent, baseStyle, "div");
```

Comment:
You importing `toReact` adapter but trying to use `reactSvelte`.
And importing `SvelteComponent` but trying to use `SvelteApp`.

Now it's synced with imports.